app/Exceptions/ValidationException.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Ushahidi\App\Exceptions;
+
+use Symfony\Component\HttpKernel\Exception\UnprocessableEntityHttpException;
+
+/**
+ * Validation Exception
+ */
+class ValidationException extends UnprocessableEntityHttpException
```

Comment:
I suppose that we are doing this to prevent error responses from breaking? Are we testing this using API tests?